### PR TITLE
Fix in-page links and shorten headings

### PR DESCRIPTION
**Pull request by mfuji09**
### Description

- Fix broken in-page links
- Shorten some verbose headings

### Motivation

Many of in-page links in this page don't work because fragments of these links aren't updated.
And there are very verbose headings which makes difficult to create links.

### CODE REVIEW

File: files/en-us/web/api/file_and_directory_entries_api/introduction/index.md [hunk 1]
```diff
@@ -74,33 +74,33 @@ The following are just a few examples of how you can use the File and Directory
...
-- [When using the asynchronous API, always use the error callbacks](#errorcallbacks)
-- [The File and Directory Entries API interacts with other APIs](#interfaces)
-- [The File and Directory Entries API is case-sensitive](#case)
+- [Virtualized file system](#virtualized_file_system)
+- [Different storage types](#different_storage_types)
+- [Storage quotas](#storage_quotas)
+- [Asynchronous and synchronous versions](#asynchronous_and_synchronous_versions)
+- [Using the error callbacks for asynchronous API](#using_the_error_callbacks_for_asynchronous_api)
+- [Interacting with other APIs](#interacting_with_other_apis)
+- [Case-sensitive](#case-sensitive)
```
Comment (teoli2003) [inline files/en-us/web/api/file_and_directory_entries_api/introduction/index.md:83]: This is a kind of summary, so not needed.
```suggestion
```

File: files/en-us/web/api/file_and_directory_entries_api/introduction/index.md [hunk 2]
```diff
@@ -74,33 +74,33 @@ The following are just a few examples of how you can use the File and Directory
 
 Before you start using the File and Directory Entries API, you need to understand a few concepts:
```
Comment (teoli2003) [inline files/en-us/web/api/file_and_directory_entries_api/introduction/index.md:75]: To match the removal of the pseudo-table of content.
```suggestion
Before using the _File and Directory Entries API_, you must understand a few concepts.
```

File: files/en-us/web/api/file_and_directory_entries_api/introduction/index.md [hunk 3]
```diff
@@ -123,30 +123,30 @@ The File and Directory Entries API is designed to be used with other APIs and el
...
 For security reasons, browsers impose restrictions on file access. If you ignore them, you will get security errors.
 
-- [The File and Directory Entries API adheres to the same-origin policy](#origin)
-- [The File and Directory Entries API does not let you create and rename executable files](#execute)
-- [The file system is sandboxed](#sandbox)
-- [You cannot run your app from file://](#file)
+- [Adhering to the same-origin policy](#adhering_to_the_same-origin_policy)
+- [Unable to create and rename executable files](#unable_to_create_and_rename_executable_files)
+- [Sandboxed file system](#sandboxed_file_system)
+- [You cannot run your app from file://](#you_cannot_run_your_app_from_file)
```
Comment (teoli2003) [inline files/en-us/web/api/file_and_directory_entries_api/introduction/index.md:137]: Removing pseudo-toc
```suggestion
```

**Review by teoli2003 — APPROVED** [review]

File: files/en-us/web/api/file_and_directory_entries_api/introduction/index.md [hunk 4]
```diff
@@ -72,35 +72,28 @@ The following are just a few examples of how you can use the File and Directory
...
-Before you start using the File and Directory Entries API, you need to understand a few concepts:
+Before using the _File and Directory Entries API_, you must understand a few concepts.
 
-- [The File and Directory Entries API is a virtual representation of a file system](#virtual)
-- [The File and Directory Entries API can use different storage types](#storage)
-- [Browsers impose storage quota](#quota)
-- [The File and Directory Entries API has asynchronous and synchronous versions](#versions)
-- [When using the asynchronous API, always use the error callbacks](#errorcallbacks)
-- [The File and Directory Entries API interacts with other APIs](#interfaces)
-- [The File and Directory Entries API is case-sensitive](#case)
```
Comment (teoli2003) [inline files/en-us/web/api/file_and_directory_entries_api/introduction/index.md:77]:
```suggestion
```

File: files/en-us/web/api/file_and_directory_entries_api/introduction/index.md [hunk 5]
```diff
@@ -123,30 +118,26 @@ The File and Directory Entries API is designed to be used with other APIs and el
...
 The filesystem API is case-sensitive, and case-preserving.
 
 ## Restrictions
 
 For security reasons, browsers impose restrictions on file access. If you ignore them, you will get security errors.
 
-- [The File and Directory Entries API adheres to the same-origin policy](#origin)
-- [The File and Directory Entries API does not let you create and rename executable files](#execute)
-- [The file system is sandboxed](#sandbox)
-- [You cannot run your app from file://](#file)
```
Comment (teoli2003) [inline files/en-us/web/api/file_and_directory_entries_api/introduction/index.md:129]:
```suggestion
```

**Comment by mfuji09** [comment]
Thank you @teoli2003 ! It looks much better.

**Pull request merged**